### PR TITLE
Jupiter Terminal Integration user guide + demo

### PR DESCRIPTION
**Pull request by benlvb**

### CODE REVIEW

**Review by dongnguyen9186 — APPROVED** [review]

**Comment by dongnguyen9186** [comment]
@benlvb  can you move images into static folder?

**Review by dongnguyen9186 — CHANGES_REQUESTED** [review]
can you move images into static folder?

**Review by Timoon21 — COMMENTED** [review]
Currently, “guides” section is “user oriented” and “docs” section is “technical docs”.

Jup terminal integration seems more a “technical docs”. Have you considered create a “tutorial” section in the docs instead?

Other than that, this guide is okay for me.

**Review by Timoon21 — COMMENTED** [review]
Currently, “guides” section is “user oriented” and “docs” section is “technical docs”.

Jup terminal integration seems more a “technical docs”. Have you considered create a “tutorial” section in the docs instead?

Other than that, this guide is okay for me.

File: guides/8-terminal-integration-guide.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+With several templates to get you started, and auto generated code snippets.
+
+**Github Repo**: https://github.com/jup-ag/terminal 
+
+## Quick tutorial with Solana Dapp Scaffold
+
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/T-3KN3k1e5Y" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+[Watch tutorial on YouTube](https://youtu.be/T-3KN3k1e5Y)
```
Comment (dicksonpys) [inline guides/8-terminal-integration-guide.md:22]: awesome, simple, short, and concise

**Review by dicksonpys — APPROVED** [review]
thanks for addressing your engineer's camera shyness, approved on my side 👍

**Review by dongnguyen9186 — APPROVED** [review]

**Pull request merged**